misc/validate-state/validate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+if [[ "$IS_REDEPLOY" == "true" ]];
+then
+  EXPECTED_PLAN_OUTPUT="Plan: 0 to add, 0 to change, 0 to destroy."
+  PLAN_OUTPUT=$(terraform show -no-color .tf-plan | tail -1)
```

Comment:
I would print the `PLAN_OUTPUT` and `EXPECTED_PLAN_OUTPUT` to the console to better debug it when there will be an issue.

Reply:
EXPECTED_PLAN_OUTPUT is static and PLAN_OUTPUT is printed when they are not equal